File: src/Introduction/HomeWorks/HW3/GetNumbers.java
package Introduction.HomeWorks.HW3;

import java.util.ArrayList;
import java.util.Random;

public class GetNumbers {
    public static ArrayList<Integer> getList(int size) {
        ArrayList<Integer> list = new ArrayList<>(size);
        Random rnd = new Random();

        for (int i = 0; i < size; i++) {
            list.add(rnd.nextInt(1,20));
        }

        return list;
    }

    public static int[] getArray(int size) {
        Random rnd = new Random();
        int[] array = new int[size];
        for (int i = 0; i < size; i++) {
            array[i] = rnd.nextInt(20);
        }
        return array;
    }
}

File: src/Introduction/HomeWorks/HW3/Kotlin/GetNumbers.kt
package Introduction.HomeWorks.HW3.Kotlin

import Introduction.HomeWorks.HW3.Java.GetNumbers.getList
import java.util.Random

class GetNumbers {

    companion object {
        fun get(size : Int) = getList(size)
    }
    fun getList(size : Int) : ArrayList<Int> {
        val list : ArrayList<Int> = ArrayList(size)
        val rnd = Random()
        for (number in 0..size) {
            list.add(rnd.nextInt())
        }
        return list
    }
}